recipes/flit/bld.bat

```diff
@@ -0,0 +1,2 @@
+python -m flit install --env --deps none
```

Comment:
Try changing this to `%PYTHON%` to ensure you are getting the right one from the build env.